docs/reference/zoom.md

```diff
@@ -0,0 +1,141 @@
+
+## Availability and prerequisites
+
+This connector is available as a **connector client** from the **Python connectors framework**. To use this connector, satisfy all [connector client requirements](https://www.elastic.co/guide/en/enterprise-search/master/build-connector.html).
+
+This connector is in **beta** and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.
+
+## Usage
+
+To use this connector as a **connector client**, use the **Zoom** tile from the connectors list OR **Customized connector** workflow.
+
+For additional operations, see [Usage](https://www.elastic.co/guide/en/enterprise-search/master/connectors-usage.html).
+
+## Connecting to Zoom
+
+To connect to Zoom you need to [create an Server-to-Server OAuth application](https://developers.zoom.us/docs/internal-apps/s2s-oauth/) that can access resources. Follow these steps:
+
+1. Go to the Zoom App Marketplace (https://marketplace.zoom.us/) and sign in with your Zoom account.
+
+2. Navigate to the "Develop" service.
+
+3. Select "Build App" from the dropdown menu.	
+
+4. Click on the "Server-to-Server OAuth" button to register a new application.
+
+5. Provide a name for your app.
+
+6. Click on the "Create" button to create the app registration.
+
+7. After the registration is complete, you will be redirected to the app's overview page. Take note of the "App Credentials" value, as you'll need it later.
+
+8. Navigate to the "Scopes" section and click on the "Add Scopes" button.
+
+9. The following scopes need to be added to the app.
+
+```shell
+user:read:admin
+meeting:read:admin
+chat_channel:read:admin
+recording:read:admin
+chat_message:read:admin
+report:read:admin
+```
+
+10. Click on the "Done" button to add the selected scopes to your app.
+
+11. Navigate to the "Activation" section and input the necessary information to activate the app.
+
+After completion, use the following configuration parameters to configure the connector.
+
+## Configuration
+
+### Configure Zoom connector
+
+The following configuration fields are required:
+
+#### `Zoom application Account ID`  (required)
+
+The "Account ID" is a unique identifier associated with a specific Zoom account within the Zoom platform, found on the app's overview page. Example:
+
+- `KVx-aQssTOutOAGrDfgMaA`
+
+#### `Zoom application Client ID`  (required)
+
+The "Client ID" refers to a unique identifier associated with an application that integrates with the Zoom platform, found on the app's overview page. Example:
+
+- `49Z69_rnRiaF4JYyfHusw`
+
+#### `Zoom application Client Secret`  (required)
+
+The "Client Secret" refers to a confidential piece of information generated when developers register an application on the Zoom Developer Portal for integration with the Zoom platform, found on the app's overview page. Example:
+
+- `eieiUJRsiH543P5NbYadavczjkqgdRTw`
```

Comment:
Double checking, this isn't a real secret that's been pasted as an "example", right?

Reply:
Yes! Not real one